cmd/terramate/cli/cli.go

```diff
@@ -804,11 +1065,17 @@ func (p *project) setDefaults(parsedArgs *cliSpec) error {
 	if baseRef == "" {
 		baseRef = gitOpt.BaseRef
 		if p.isRepo {
+			logger.Trace().
+				Str("configFile", p.root+"/terramate.tm.hcl").
+				Msg("Create new git wrapper.")
 			gw, err := newGit(p.wd, false, false)
 			if err != nil {
 				return err
 			}
 
+			logger.Trace().
+				Str("configFile", p.root+"/terramate.tm.hcl").
```

Comment:
```suggestion
				Str("configFile", filepath.Join(p.root, "terramate.tm.hcl")).
```

+ Extract the configFile field on a logger, seems repeated multiple times here.